incubator/sparkoperator/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 name: sparkoperator
 description: A Helm chart for Spark on Kubernetes operator
-version: 0.3.1
-appVersion: v2.4.0-v1beta1-0.9.0
+version: 0.3.2
```

Comment:
Let's change the version to 0.4.0 because it's a new release, not a bug fix.

Reply:
Done.